lib/StripeObject.php

```diff
@@ -27,13 +27,29 @@ public static function init()
     {
         self::$permanentAttributes = new Util\Set(array('_opts', 'id'));
         self::$nestedUpdatableAttributes = new Util\Set(array(
-            'metadata', 'legal_entity', 'address', 'dob', 'payout_schedule', 'transfer_schedule', 'verification',
-            'tos_acceptance', 'personal_address', 'personal_address_kana', 'personal_address_kanji',
-            'address_kana', 'address_kanji', 'shipping',
-            // will make the array into an AttachedObject: weird, but works for now
-            'additional_owners', 0, 1, 2, 3, 4, // Max 3, but leave the 4th so errors work properly
+            // Numbers are in place for indexes in an `additional_owners` array.
+            //
+            // There's a maximum allowed additional owners of 3, but leave the
+            // 4th so errors work properly.
+            0, 1, 2, 3, 4,
+
+            'additional_owners',
+            'address',
+            'address_kana',
+            'address_kanji',
+            'dob',
             'inventory',
+            'legal_entity',
+            'metadata',
+            'payout_schedule',
             'owner',
```

Comment:
this should be a line above

Reply:
Oops. Absolutely right! Amended in bdbfe87.

Good to go otherwise?